### ISSUE TITLE
R-devel on Linux now needs checkbashisms

### ISSUE THREAD

**Issue by nealrichardson** [r-hub/rhub-linux-builders#46]
On `Fedora Linux, R-devel, clang, gfortran`, I got

```
* checking top-level files ... WARNING
A complete check needs the 'checkbashisms' script
```

This seems to be related to https://github.com/wch/r-source/commit/0866fd0d4a2cefb3547717d13a08cf63a7e996bb, added a couple of weeks ago, which (FWIW) is triggering this NOTE on CRAN's checks for me:

```
Version: 0.15.0
Check: top-level files
Result: NOTE
    ‘configure’: /bin/bash is not portable
Flavors: r-devel-linux-x86_64-fedora-clang, r-devel-linux-x86_64-fedora-gcc
```

**Comment by maelle** [comment]
@gaborcsardi I'd like to help but I don't know how exactly. :-)

* Is adding the devscripts package (that contains the script, on Ubuntu and Debian) a good idea or should the checkbashims script only be downloaded from https://sourceforge.net/projects/checkbaskisms/files/?

* Should the script be added to the devel configurations only or in the base configurations? Or in the base configurations only once the checkbashisms stuff makes it to R release?

I found https://travis-ci.community/t/new-warning-in-r-devel-about-the-checkbashisms-script/5551/3

**Comment by maelle** [comment]
Also relevant https://github.com/rocker-org/rocker/commit/1749e1643bf37510d0e75d733c621c0e3e3f7f89 :-)

**Comment by gaborcsardi** [comment]
I would copy the file to the base containers, ubuntu, debian, fedora, etc.

**Comment by maelle** [comment]
Ok so the same approach as https://github.com/rocker-org/rocker then?

**Comment by gaborcsardi** [comment]
yeah

**Comment by maelle** [comment]
I'm still locally building the first image I amended.

For info for Travis R now devscripts is installed https://github.com/travis-ci/travis-build/commit/5ec0319d89b469e5c3afee01348b163359450d1e#diff-5b7e1aa01f1c6f058df6e0b9e2486c62

**Comment by gaborcsardi** [comment]
devscripts has tons of dependencies that we don't need. And we also need to add it to fedora, centos, etc.

**Comment by maelle** [comment]
yes, it makes sense.

I am re-building the image because I had put COPY after RUN :see_no_evil:

**Comment by nealrichardson** [comment]
Solved by #47.
